src/docfx/lib/github/GitHubAccessor.cs

```diff
@@ -96,5 +98,33 @@ public async Task<(Error, string login)> GetLoginByCommit(string repoOwner, stri
                 return (Errors.GitHubApiFailed(apiDetail, ex), null);
             }
         }
+
+        public async Task<(Error, T)> RetryApi<T>(string api, Func<Task<T>> action)
```

Comment:
We also have retry logic within `HttpClientUtility`, which is quite similar to this one, can we abstract the retry logic somewhere to be reused?

Reply:
Updated. My upcoming PR will update HttpClientUtility to use this soon.